x/sequencer/keeper/grpc_query_sequencers_by_rollapp.go

```diff
@@ -83,7 +93,8 @@ func (k Keeper) Proposers(c context.Context, req *types.QueryProposersRequest) (
 	pageRes, err := query.Paginate(sequencerStore, req.Pagination, func(key []byte, value []byte) error {
 		proposer, err := k.RealSequencer(ctx, string(value))
 		if err != nil {
-			return err
+			// skip sentinel proposers
+			return nil
```

Comment:
I think this will cause an invalid count (i.e +1 for each nil, though there isn't really another one)

Reply:
seems fine for me. count used to break when limit reached.